solar_system.rb

```diff
@@ -0,0 +1,157 @@
+# classes
+class Planet
+
+  attr_reader :name, :moons, :distance_fromsun, :mass, :orbit_period
+
+  def initialize(name, moons, distance_fromsun, mass, orbit_period)
+    @name = name
+    @moons = moons
+    @distance_fromsun = distance_fromsun
+    @mass = mass
+    @orbit_period = orbit_period
+  end
+
+# The method attributes returns the information of the planet selected by the user in the interface
+  def attributes
+    return "\n Name: #{@name}\n Moons: #{@order}\n Distance from the Sun: #{@distance_fromsun}\n Mass: #{@diameter}\n Orbit Period = #{@orbit_period}\n "
+  end
+
+end
+
+class SolarSystem
+
+  attr_accessor  :planets
+
+  def initialize(planets)
+    @planets = planets
+  end
+
+  def add_planet(planet)
+    @planets.push(planet)
+  end
+
+  def list_planets
+    list = ""
+    @planets.each_with_index do |planet, i|
+      list += "#{i+1} #{planet.name}\n"
+    end
+    return list
+  end
+
+end
+
+# Instances
+planet_A = Planet.new('Mercury','0','57,909,227 km','38', '88 days')
+planet_B = Planet.new('Venus', '0','108,209,475 km','95', '225 days')
+planet_C = Planet.new('Earth','1','149,598,262 km','211','365.24 days')
+planet_D = Planet.new('Mars','2','227,943,824 km ', '408','1.9 years')
+planet_E = Planet.new('Jupiter','1','778,340,821 km ','38','11.9 years')
+
+planets = [planet_A, planet_B, planet_C, planet_D, planet_E]
+system_ = SolarSystem.new(planets)
+
+# Method for the user to create a new planet
+def new_planet(solar)
+
+  # The loop will enable the user to continue adding new planets until wanted.
```

Comment:
I like that you've broken this and `read_about_planet` out as separate methods. Good organization!